lib/config.js

```diff
@@ -25,3 +25,8 @@ cfg.routes = {
   registration: '/authn/token/client/registration',
   recovery: '/authn/token/recovery'
 };
+
+cfg.fakeTokenOptions = {
+  jitter: 0, // randomized delay value in ms
+  hmacKey: 'exampleKey'
```

Comment:
This is the sort of new config that no one will know to set.  Should this be added to our "ensure" config setup?  (I'm forgetting how this works, needs to be tested.)
```js
config.ensureConfigOverride.fields.push(
  'authn-token-http.fakeTokenOptinos.hmacKey'
);
```

Reply:
For numerous reasons `ensureConfigOverride` is a broken feature and is not being used in practice.  It is however *very* important that a setting like this be well documented in the changelog etc.